components/ws-manager/pkg/manager/monitor.go

```diff
@@ -959,6 +966,12 @@ func (m *Monitor) finalizeWorkspaceContent(ctx context.Context, wso *workspaceOb
 		}
 		break
 	}
+	wsType := api.WorkspaceType_name[int32(tpe)]
+	hist, errHist := m.manager.metrics.finalizeTimeHistVec.GetMetricWithLabelValues(wsType)
```

Comment:
This may be an elementary question, but why not use `err` here?

Reply:
you are right, that was copy paste mistake. fixed!